README.md

```diff
@@ -29,15 +29,15 @@ Add the following dependency to your pom.xml:
 ### Using with sbt
 
 ```
-libraryDependencies += "com.intel.chimera" % "chimera" % "0.9.0"
+libraryDependencies += "org.apache.commons.crypto" % "chimera" % "0.9.0"
 ```
 
 ## Usage 
 
 ```java
 
 Properties properties = new Properties();
-properties.setProperty("chimera.crypto.cipher.classes", "com.intel.chimera.crypto.OpensslCipher");
+properties.setProperty("chimera.crypto.cipher.classes", "org.apache.commons.crypto.crypto.OpensslCipher");
```

Comment:
The configuration name has "chimera" in it. Create another JIRA if we don't want to address it here.